README.md

```diff
@@ -114,6 +114,11 @@ cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data setup
 
 ```
 
+Setup device with generation of TLS certificates:
+```
+cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data setup --url http://127.0.0.1:8000/ --generate-device-tls-certificate
```

Comment:
The first section of #usage is missing the new flag.

Reply:
+1